content/luau/concepts/tables/tables.md

```diff
@@ -0,0 +1,76 @@
+---
+Title: 'Tables'
+Description: 'luau is a scripting language particularly for Roblox game development. This document is meant to demonstrate the use case for luau tables.'
+Codecademy Hub Page: 'https://www.codecademy.com/catalog/language/luau'
+CatalogContent:
+  - 'paths/computer-science'
+Subjects:
+  - 'Developer Tools'
+  - 'Game Development'
+Tags:
+  - 'Markdown'
+  - 'GitHub'
+  - 'Documentation'
+---
+
+The table data type can store multiple values of any type that isn't nil, including booleans, numbers, strings, functions, and other tables.
+
+The syntax for an empty table is:
+
+```
+local table = {}
+```
+
+In the context of Lua, the local keyword is primarily used to declare local variables. It is not used specifically for instantiating tables. You can use the local keyword to create local variables of various types, including tables, numbers, strings, functions, and more. What creates the table is the curly braces in the syntax ({}). If you wanted to instantiate a table with parameters you can write something like this:
```

Comment:
[proofreading] Since tables are being discussed in a Luau context, consider using that instead of Lua. Also, hyperlink the word to help readers see the connection more. 
```suggestion
In the context of [Luau](https://www.codecademy.com/resources/docs/luau), the local keyword is primarily used to declare local variables. It is not used specifically for instantiating tables. You can use the local keyword to create local variables of various types, including tables, numbers, strings, functions, and more. What creates the table is the curly braces in the syntax ({}). If you want to instantiate a table with parameters you can write something like this:
```